sources/platform/actors/development/actor_definition/input_schema/index.md

```diff
@@ -0,0 +1,80 @@
+    "properties": {
+        "startUrls": {
+            "title": "Start URLs",
+            "type": "array",
+            "description": "One or more URLs of pages where the crawler will start. Note that the Actor will additionally only crawl sub-pages of these URLs. For example, for start URL `https://www.example.com/blog`, it will crawl pages like `https://example.com/blog/article-1`, but will skip `https://example.com/docs/something-else`.",
+            "editor": "requestListSources",
+            "prefill": [{ "url": "https://docs.apify.com/" }]
+        },
+        "crawlerType": {
+            "sectionCaption": "Crawler settings",
+            "title": "Crawler type",
+            "type": "string",
+            "enum": ["playwright:chrome", "cheerio", "jsdom"],
+            "enumTitles": ["Headless web browser (Chrome+Playwright)", "Raw HTTP client (Cheerio)", "Raw HTTP client with JS execution (JSDOM) (experimental!)"],
+            "description": "Select the crawling engine:\n- **Headless web browser** (default) - Useful for modern websites with anti-scraping protections and JavaScript rendering. It recognizes common blocking patterns like CAPTCHAs and automatically retries blocked requests through new sessions. However, running web browsers is more expensive as it requires more computing resources and is slower. It is recommended to use at least 8 GB of RAM.\n- **Raw HTTP client** - High-performance crawling mode that uses raw HTTP requests to fetch the pages. It is faster and cheaper, but it might not work on all websites.",
+            "default": "playwright:chrome"
+        },
+        "maxCrawlDepth": {
+            "title": "Max crawling depth",
+            "type": "integer",
+            "description": "The maximum number of links starting from the start URL that the crawler will recursively descend. The start URLs have depth 0, the pages linked directly from the start URLs have depth 1, and so on.\n\nThis setting is useful to prevent accidental crawler runaway. By setting it to 0, the actor will only crawl start URLs.",
+            "minimum": 0,
+            "default": 20
+        },
+        "maxCrawlPages": {
+            "title": "Max pages",
+            "type": "integer",
+            "description": "The maximum number pages to crawl. It includes the start URLs, pagination pages, pages with no content, etc. The crawler will automatically finish after reaching this number. This setting is useful to prevent accidental crawler runaway.",
+            "minimum": 0,
+            "default": 9999999
+        },
+        ...
+    }
+}
+```
+
+The actual input object passed from the autogenerated input UI to the Actor then looks as follows:
+
+```json
+{
+    "debugMode": false,
+    "proxyConfiguration": {
+        "useApifyProxy": true
+    },
+    "saveHtml": false,
+    "saveMarkdown": false,
+    "saveScreenshots": false,
+    "startUrls": [
+        {
+            "url": "https://docs.apify.com/"
+        }
+    ]
+}
+```
```

Comment:
It ends kinda abruptly here. Would be good to add a paragraph telling me that there are other chapters related to input schema and what they will discuss.